pages/dashboard.vue

```diff
@@ -184,4 +200,12 @@ export default Vue.extend ({
   font-size: $px-18;
   padding-top: 50px;
 }
+
+.service-container {
+  cursor: pointer;
+
+  &:hover {
+    box-shadow: 1px 1px 6px 0px $gray6;
+  }
```

Comment:
Was box shadow called out in the Visual Design specs?

Reply:
yes, I think this is what the spec wants.
![image](https://user-images.githubusercontent.com/62360407/168320446-8e1d5beb-a9a2-4819-b61f-dad56593c7cd.png)

Reply:
Ok cool, thanks for verifying 👍

Reply:
Resolved (sorry don't have permissions here to do it :P)